### PR TITLE
Added VSCode support to .gitignore

### PR DESCRIPTION
**Pull request by GabrielRavier**
This is for VS Code users (such as me) obviously, so this won't have any observable effects on anybody else

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,12 @@ msvc2003/Debug
...
 msvc2003/Release (Japanese)
+
+# Exclude VSCode folder
+.vscode/*
+
+# Do not exclude common files that we might actually add in the future (could be used for having build commands or the like)
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
```
Comment (Clownacy) [inline .gitignore]: I don't plan on adding VSCode files so these last lines might as well be removed.

Comment (GabrielRavier) [reply .gitignore]: Okay, I'll just comment them out in case we need them in the future (and for those that may need them)

Comment (GabrielRavier) [reply .gitignore]: Tbh I think I'd rather keep them so anyone making those files (such as me) doesn't end up accidentally excluding them

Comment (GabrielRavier) [reply .gitignore]: Just so yunno I got those from [the official Github .gitignore repo](https://github.com/github/gitignore/blob/master/Global/VisualStudioCode.gitignore)

**Pull request merged**